### PR TITLE
Feat/dtrsd 79696: improve default payment method section

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/artemis-batch-2`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTRSD-79696
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. improve default payment section

## Related
- [x] TRANS-43400

### CODE REVIEW

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/new/payment/components/default/index.html [hunk 1]
```diff
@@ -4,27 +4,55 @@
...
+            </a>
+        </p>
+
+        <!--Default payment method-->
+        <div class="p-3 border border-info rounded d-flex align-items-center justify-content-between">
+
+            <!--Logo-->
+            <img data-ng-if="$ctrl.defaultPaymentMethod.icon.data"
+                 data-ng-src="{{ $ctrl.defaultPaymentMethod.icon.data }}"
+                 alt=""
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/new/payment/components/default/index.html:35]: **question:** Can we add some text to `alt`?

**Review by anooparveti — COMMENTED** [review]
Since only `fr_FR` are modified, you can tick the appropriate check box.

File: packages/manager/modules/pci/src/projects/new/payment/components/default/controller.js [hunk 2]
```diff
@@ -6,4 +8,12 @@ export default class PciProjectNewPaymentDefaultCtrl {
       myAccount: { application: 'dedicated', path: '#/useraccount/dashboard' },
     });
   }
+
+  getFormatCreditCardNumber() {
+    return `····${this.defaultPaymentMethod.label.replace(/\D/g, '')}`;
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/new/payment/components/default/controller.js:13]: Hmm should you not get only the last credit card number here?

File: packages/manager/modules/pci/src/projects/new/payment/components/default/index.html (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/new/payment/components/default/index.html:35]: The `img` element was already here.
I juste retake it.

I will ask to know what we can add.

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/new/payment/components/default/index.html]: Done

File: packages/manager/modules/pci/src/projects/new/payment/components/default/controller.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/new/payment/components/default/controller.js]: Ah good point. 

Done

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/new/payment/components/default/index.html [hunk 3]
```diff
@@ -4,27 +4,56 @@
...
-        <strong
-            class="text-success d-inline-block vertical-middle"
-            data-translate="pci_project_new_payment_default_method"
-        >
-        </strong>
+
+        <!--Go to payment section-->
+        <p class="text-right">
+            <a data-ng-href="{{::$ctrl.sectionUrls.payment}}"
+               target="_blank"
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/new/payment/components/default/index.html:18]: No need for a rel=noopener ?

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/new/payment/components/default/index.html:20]: it's redirect to our Manager.
Do you think it's a security leaks ?

**Review by anooparveti — APPROVED** [review]

**Comment by marie-j** [comment]
Hello @mohammed-zahaf 
Could you please rebase ? Looks like some commits don't belong to this PR 
Thanks

**Comment by mohammed-zahaf** [comment]
> Hello @mohammed-zahaf Could you please rebase ? Looks like some commits don't belong to this PR Thanks

Hello, rebased done

**Pull request merged**